bin/ksql-run-class

```diff
@@ -3,20 +3,16 @@
 
 # Use shellcheck to lint this file
 
-set -ue
-
-#cd -P deals with symlink from /bin to /usr/bin
 base_dir=$( cd -P "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
 
 # Log directory to use
-if [ -z "${LOG_DIR:-}" ]; then
-  LOG_DIR="/tmp/ksql-logs"
+if [ "x$LOG_DIR" = "x" ]; then
+  LOG_DIR="$base_dir/logs"
 fi
 
 # create logs directory
-if [[ -n $LOG_DIR ]]; then
-    [[ -d $LOG_DIR ]] || mkdir -p "$LOG_DIR"
-#    KSQL_LOG4J_OPTS="-Dksql.log.dir=$LOG_DIR ${KSQL_LOG4J_OPTS}"
+if [ ! -d "$LOG_DIR" ]; then
+  mkdir -p "$LOG_DIR"
```

Comment:
I can't add a comment inline, but consider adding `monitoring-interceptors` in the classpath below as: 

`for library in "confluent-common" "rest-utils" "ksql-engine" "ksql-rest-app" "ksql-cli" "ksql" "monitoring-interceptors"; do`

where now we have: 
`for library in "confluent-common" "rest-utils" "ksql-engine" "ksql-rest-app" "ksql-cli" "ksql"; do`

Reply:
Done.